src/nvim/edit.c

```diff
@@ -3794,12 +3794,9 @@ bool in_cinkeys(int keytyped, int when, bool line_is_empty)
       while (*look == '>') {
         look++;
       }
-    }
-    /*
-     * Is it a word: "=word"?
-     */
-    else if (*look == '=' && look[1] != ',' && look[1] != NUL) {
-      ++look;
+    } else if (*look == '=' && look[1] != ',' && look[1] != NUL) {
+      // Is it a word: "=word"?
```

Comment:
This is clearer about what the comment refers to
```suggestion
      // Is it a word: "=word"?
    } else if (*look == '=' && look[1] != ',' && look[1] != NUL) {
```

Reply:
I kind of disagree on this one, and the suggestion here goes against the convention we've been applying when fixing these lint errors.